src/WebJobs.Script/Extensions/BindingMetadataExtensions.cs

```diff
@@ -1,12 +1,53 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
 using Microsoft.Azure.WebJobs.Script.Description;
 
 namespace Microsoft.Azure.WebJobs.Script.Extensions
 {
     public static class BindingMetadataExtensions
     {
+        private const string HttpTriggerKey = "HttpTrigger";
+
+        private static readonly HashSet<string> DurableTriggers = new(StringComparer.OrdinalIgnoreCase)
+        {
+            "EntityTrigger",
+            "ActivityTrigger",
+            "OrchestrationTrigger"
```

Comment:
Is `durableClient` a valid type value to be checked againist?

Reply:
No, we only care about triggers. Client is an input binding.